Makefile

```diff
@@ -22,22 +22,22 @@ console:
 
 build-latest:
 	$(MAKE) build TAG=latest \
-		XDR_REF=v20.1.0 \
-		CORE_REF=v20.4.0 \
+		XDR_REF=v21.0.0 \
```

Comment:
just curious, why are you changing these other image variants here for new versions but not on the GH CI builds? i'm asking b/c would follow this same pattern for bumping the new rpc version as well if this is the process.

Reply:
That's a good point - I just followed a similar PR that did the bumps. IDK why it didn't update all the actions; I've pushed the change to update `build-latest.yml`. I don't see any other places where the old version is being used.

Reply:
rc releases are updated on testing only usually, not latest and future.

Reply:
Thanks for letting us know, reverted.

Reply:
Ok, I've updated only testing. build-future is outdated though, not sure if this one is actually being used properly.